oxidation/bindings/generator/templates/binding_web_meths.rs.j2

```diff
@@ -138,9 +138,9 @@ JsValue::from(Uint8Array::from({{ rs_value }}.as_ref()))
 {%- elif type.kind == "list" -%}
 {
     let js_array = Array::new_with_length({{ rs_value }}.len() as u32);
-    for elem in {{ rs_value }} {
+    for (i, elem) in {{ rs_value }}.into_iter().enumerate() {
         let js_elem = {{ render_rs_to_js_value("elem", type.elem) | indent(8) }};
-        js_array.push(&js_elem);
+        js_array.set(i as u32, js_elem);
```

Comment:
Because according to: https://rustwasm.github.io/wasm-bindgen/api/js_sys/struct.Array.html#method.new_with_length
`new_with_length` allocate with `undefined element`, so we have to replace `undefined value`

Reply:
I see 😄 
a comment to explain this would be good

Reply:
Should I add the comment in `template` ?

Reply:
yup (there is already some `// TODO: ...` in the template if I recall well)

Reply:
Done